### ISSUE TITLE
PyTest Warning

### ISSUE THREAD

**Issue by simon-mo** [modin-project/modin#79]
```
/usr/local/lib/python2.7/site-packages/_pytest/python.py:197: RemovedInPytest4Warning: Fixture "test_ndim" called directly. Fixtures are not meant to be called directly, are created automatically when test functions request them as parameters. See https://docs.pytest.org/en/latest/fixture.html for more information.
```

Our CI is giving us tons of these kind of warning. Worth investigating.

**Comment by thomasjpfan** [comment]
This comes from the fact that functions decorated with `@pytest.fixture` were designed to be passed in as parameters:

```python
@pytest.fixture
def one():
    return 1

def test_simple(one):
    assert one == 1
```

In the codebase there are fixtures such as:

```python
@pytest.fixture
def test_copy(ray_df):
    ....

def test_mixed_dtype_dataframe():
    ....
    test_copy(ray_df)
```

One option to refactor this would be:

```python
def assert_copy(ray_df):
    ....

def test_mixed_dtype_dataframe():
    ....
    assert_copy(ray_df)
```

**Comment by devin-petersohn** [comment]
@thomasjpfan Seems like a reasonable refactor. Once #70 gets merged, this can be done more effectively (and without rebasing issues).

**Comment by devin-petersohn** [comment]
Resolved by #903
